ci/cpu/prebuild.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-#Build rmm once per PYTHON
-if [[ "$CUDA" == "10.1" ]]; then
+#Build rmm once per PYTHON-CUDA combo
+if [[ "$PYTHON" == "3.7" ]] && [[ "$CUDA" == "10.1" ]]; then
     export UPLOAD_RMM=1
 else
     export UPLOAD_RMM=0
```

Comment:
This should just always set `UPLOAD_RMM=1`, you don't want to only upload when Python is 3.7 and CUDA is 10.1